packages/zoe/test/unitTests/contracts/test-atomicSwap.js

```diff
@@ -1,144 +1,171 @@
 
 test('zoe - atomicSwap', async t => {
-  t.plan(11);
-  const {
-    moolaIssuer,
-    simoleanIssuer,
-    moolaMint,
-    simoleanMint,
-    moola,
-    simoleans,
-  } = setup();
-  const zoe = makeZoe(fakeVatAdmin);
-  const inviteIssuer = zoe.getInviteIssuer();
-
-  // pack the contract
-  const bundle = await bundleSource(atomicSwapRoot);
-  // install the contract
-  const installationHandle = await zoe.install(bundle);
-
-  // Setup Alice
-  const aliceMoolaPayment = moolaMint.mintPayment(moola(3));
-  const aliceMoolaPurse = moolaIssuer.makeEmptyPurse();
-  const aliceSimoleanPurse = simoleanIssuer.makeEmptyPurse();
-
-  // Setup Bob
-  const bobSimoleanPayment = simoleanMint.mintPayment(simoleans(7));
-  const bobMoolaPurse = moolaIssuer.makeEmptyPurse();
-  const bobSimoleanPurse = simoleanIssuer.makeEmptyPurse();
-
-  // 1: Alice creates an atomicSwap instance
-  const issuerKeywordRecord = harden({
-    Asset: moolaIssuer,
-    Price: simoleanIssuer,
-  });
-  const { invite: aliceInvite } = await zoe.makeInstance(
-    installationHandle,
-    issuerKeywordRecord,
-  );
-
-  // 2: Alice escrows with zoe
-  const aliceProposal = harden({
-    give: { Asset: moola(3) },
-    want: { Price: simoleans(7) },
-    exit: { onDemand: null },
-  });
-  const alicePayments = { Asset: aliceMoolaPayment };
-
-  // 3: Alice makes the first offer in the swap.
-  const { payout: alicePayoutP, outcome: bobInviteP } = await zoe.offer(
-    aliceInvite,
-    aliceProposal,
-    alicePayments,
+  t.plan(8);
+  const { moolaKit, simoleanKit, moola, simoleans, zoe } = setup();
+
+  const makeAlice = async moolaPayment => {
+    const moolaPurse = await E(moolaKit.issuer).makeEmptyPurse();
+    const simoleanPurse = await E(simoleanKit.issuer).makeEmptyPurse();
+    return {
+      installCode: async () => {
+        // pack the contract
+        const bundle = await bundleSource(atomicSwapRoot);
+        // install the contract
+        const installationP = E(zoe).install(bundle);
+        return installationP;
+      },
+      makeInstance: async installation => {
+        const issuerKeywordRecord = harden({
+          Asset: moolaKit.issuer,
+          Price: simoleanKit.issuer,
+        });
+        const adminP = zoe.makeInstance(installation, issuerKeywordRecord);
+        return adminP;
+      },
+      offer: async firstInvitation => {
+        const proposal = harden({
+          give: { Asset: moola(3) },
+          want: { Price: simoleans(7) },
+          exit: { onDemand: null },
+        });
+        const payments = { Asset: moolaPayment };
+
+        const seat = await E(zoe).offer(firstInvitation, proposal, payments);
+
+        E(seat)
+          .getPayout('Asset')
+          .then(moolaPurse.deposit)
+          .then(amountDeposited =>
+            t.deepEquals(
+              amountDeposited,
+              moola(0),
+              `Alice didn't get any of what she put in`,
+            ),
```

Comment:
This might be an improvement on `assertPayout` which I added in zoeTestHelpers. Rather than adding 9 lines of code in several places, what do you think of calling/fixing `assertPayout()`?